src/components/Microsoft.DotNet.UpgradeAssistant/TargetFramework/WindowsSdkTargetFrameworkSelectorFilter.cs

```diff
@@ -35,12 +33,12 @@ private static bool TryGetMoniker(ITargetFrameworkSelectorFilterState updater, [
             // Projects with Windows Desktop components or a WinExe output type should use a -windows suffix
             if (updater.Components.HasFlag(ProjectComponents.WindowsDesktop) || updater.Project.OutputType == ProjectOutputType.WinExe)
             {
-                tfm = $"{current}{WindowsSuffix}";
+                tfm = current with { Platform = TargetFrameworkMoniker.Platforms.Windows };
 
                 if (updater.Components.HasFlag(ProjectComponents.WinRT))
                 {
                     // TODO: Default to this version to ensure everything is supported.
-                    tfm += "10.0.19041.0";
+                    tfm = tfm with { PlatformVersion = TargetFrameworkMoniker.Net50_Windows_10_0_19041_0.PlatformVersion };
```

Comment:
i guess this is what i asked for :D but my intent was that your comment declares this is a default. seems like we should have a `PlatformVersion = TargetFrameworkMoniker.Net50_DefaultWindowsPlatform`

Reply:
I'll plan on leaving this as-is as it's not really a "DefaultWindowsPlatform" but a default we're using for this specific usage due to simplicity